packages/kmod-5.10-nvidia/nvidia-tesla-path.env.in

```diff
@@ -0,0 +1 @@
+NVIDIA_PATH=/usr/bin/nvidia/tesla/NVIDIA_VERSION/
```

Comment:
nit: can omit trailing slash, might also benefit from not hardcoding the expected path